src/Domain/SubscriptionRepository.php

```diff
@@ -11,4 +11,5 @@
  */
 interface SubscriptionRepository {
 	public function storeSubscription( Subscription $subscription );
+	public function countSimilar( Subscription $subscription, \DateInterval $interval ): int;
```

Comment:
A comment here that details the contract of the method would be nice. As is, I need to look at one of the concrete implementations to get a better understanding.